templates/index.html

```diff
@@ -1,13 +1,180 @@
+        </div>
+      </a>
+    </div>
+    <div class="col-3">
+      <a href="/wordpress" class="p-card p-button">
+        <div class="u-clearfix">
+          <div class="u-float-left">
+            <img src="https://api.jujucharms.com/charmstore/v5/kafka-51/icon.svg" alt="" class="p-card__thumbnail">
+          </div>
+        </div>
+        <h5 class="p-card__title u-no-margin--bottom">Kafka</h5>
+        <div class="p-card__content">
+          <p class="u-text--subtle">by BigData Charmers</p>
+          
+          <p><small>Kafka is a high-performance, scalable, distributed messaging...</small></p>
+        </div>
+      </a>
+    </div>
+    <div class="col-3">
+      <a href="/wordpress" class="p-card p-button">
+        <div class="u-clearfix">
+          <div class="u-float-left">
+            <img src="https://api.jujucharms.com/charmstore/v5/elasticsearch-43/icon.svg" alt="" class="p-card__thumbnail">
+          </div>
+        </div>
+        <h5 class="p-card__title u-no-margin--bottom">Elasticsearch</h5>
+        <div class="p-card__content">
+          <p class="u-text--subtle">by Llama Charmers</p>
+
+          <p><small>Distributed RESTful search and analytics.</small></p>
+        </div>
+      </a>
+    </div>
+    <div class="col-3">
+      <a href="/wordpress" class="p-card p-button">
+        <div class="u-clearfix">
+          <div class="u-float-left">
+            <img src="https://api.jujucharms.com/charmstore/v5/prometheus-7/icon.svg" alt="" class="p-card__thumbnail">
+          </div>
+        </div>
+        <h5 class="p-card__title u-no-margin--bottom">Prometheus</h5>
+        <div class="p-card__content">
+          <p class="u-text--subtle">by Prometheus Charmers</p>
+
+          <p><small>Prometheus is a systems and service monitoring system.</small></p>
+        </div>
+      </a>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <a href="/store" class="p-button">Browse the collection</a>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2 class="p-heading--3">Pure Python Operators</h2>
+    <h3 class="p-heading--4">Python rocks for DevSecOps</h3>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p>Take “infrastructure as code” to the next level.</p>
+      <p>Control all your K8s workloads with Python, a high-level language with a huge community. Reuse open-source Python libraries for all your integration. Get standard libraries for standard workloads such as MySQL, Postgres, Etcd, Vault, MongoDB, Cassandra, Prometheus, Kafka, Rsyslog, RabbitMQ and Elastic.</p>
+      <p>The Python Operator Framework supports Linux and Windows, on x86 and ARM.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2 class="p-heading--3">Universal Operator Framework</h2>
+    <h3 class="p-heading--4">Operators for legacy applications on VMs, clouds, and bare metal. YES.
```

Comment:
```suggestion
    <h3 class="p-heading--4">Operators for legacy applications on VMs, clouds, and bare metal. YES.</h3>
```
Missing closing `</h3>`